ztp/resource-generator/tools/patchImageReference.sh

```diff
@@ -7,7 +7,7 @@ set -e
 
 BASEDIR=$1
 REPLACEMENT_IMAGE=$2
-UPSTREAM_IMAGE="quay.io/redhat_emp1/ztp-site-generator:latest"
+UPSTREAM_IMAGE="quay.io/openshift-kni/ztp-site-generator:4.10.0"
```

Comment:
just want to make sure we should refernce the 4.10.0 tag here instead of latest right?

Reply:
Is it `4.10.0` or `v4.10.0`?

Reply:
I looked up the existing tags here:
https://quay.io/repository/openshift-kni/ztp-site-generator?tab=tags and seems like it will 4.10.0 instead of v4.10.0.. as that is the format for the 4.11.0 tags.

Reply:
Just checked!  `4.10.0` is correct.

Reply:
Aha - But this needs to match the string that's currently in ztp/gitops-subscriptions/argocd/deployment/argocd-openshift-gitops-patch.json 

Right now it's `quay.io/redhat_emp1/ztp-site-generator:latest`, but we should probably edit THAT to match the new upstream to be `quay.io/openshift-kni/ztp-site-generator:4.10.0`

So do a quick search for `quay.io/redhat_emp1/ztp-site-generator:latest` under ztp/ and replace it with `quay.io/openshift-kni/ztp-site-generator:4.10.0` in this PR, and I think we're good to go.